fenix/app/src/main/res/values/strings.xml

```diff
@@ -2188,9 +2188,11 @@
     <!-- Clickable text for warning card informing the user that the current product is currently not available. Clicking this should inform the server that the product is available. -->
     <string name="review_quality_check_product_availability_warning_action_2" tools:ignore="UnusedResources">Report product is in stock</string>
     <!-- Title for warning card informing the user that the current product's re-analysis is still processing. -->
-    <string name="review_quality_check_reanalysis_in_progress_warning_title">Checking review quality</string>
+    <string name="review_quality_check_reanalysis_in_progress_warning_title" moz:RemovedIn="122">Checking review quality</string>
     <!-- Title for warning card informing the user that the current product's analysis is still processing. -->
-    <string name="review_quality_check_analysis_in_progress_warning_title">Checking review quality</string>
+    <string name="review_quality_check_analysis_in_progress_warning_title" moz:RemovedIn="122">Checking review quality (%s)</string>
```

Comment:
should the one being removed remain `Checking review quality` without `%s`

Reply:
Since they both were the same strings with the same comment and are used when analysing/reanalysing, thought of just using the new one in two places. Both now show the percentage progress.

Reply:
Oh you're soo right! I just re read the comment!